src/components/Pivot/Pivot.tsx

```diff
@@ -117,17 +112,37 @@ export class Pivot extends React.Component<IPivotProps, IPivotState> {
                 className={classNames(pivotLinkClassName, { 'is-disabled': link.disabled })}
                 onClick={this._onLinkClick.bind(this, itemKey)}
                 role="tab">
-                {link.linkIcon &&
-                    <Icon iconName={link.linkIcon} className={'pivot-icon'} title={link.linkText} />
-                }
-                {!link.linkIcon &&
-                    <span className={'pivot-text'} style={textStyle}>{link.linkText}</span>
-                }
+                {this._linkRenderer(link)}
                 {countText}
             </a>
         );
     }
 
+    private _linkRenderer(link: IPivotItemProps): JSX.Element {
+        const { textSize } = this.props;
+        let textStyle;
+        if (textSize) {
+            textStyle = {fontSize: textSize};
+        }
+        const textElement = <span className={'pivot-text'} style={textStyle}>{link.linkText}</span>;
+        const iconElement = <Icon iconName={link.linkIcon} className={'pivot-icon'} title={link.linkText} />;
```

Comment:
it bugs me a little that an icon element is created if i do not specify a linkIcon. But i will not hold it againts this PR.